pyproject.toml

```diff
@@ -0,0 +1,44 @@
+[project]
+name = "pixi-cuda-environment"
+version = "0.1.0"
+description = "Add a short description here"
+authors = [{name = "Eric Ma", email = "ericmajinglong@gmail.com"}]
+requires-python = ">= 3.11"
+dependencies = ["pandas>=2.2.2,<2.3", "numpy>=2.0.1,<2.1"]
+
+[build-system]
+requires = ["setuptools"]
+build-backend = "setuptools.build_meta"
+
+[tool.pixi.project]
+channels = ["conda-forge"]
+platforms = ["osx-arm64", "linux-64"]
```

Comment:
Why not also osx-64?

Reply:
For the labs project env we can put that in there. For this pedagogical repo, however, I'll have a comment saying that one can support Intel Macs by adding `osx-64`.

Reply:
I don't want to encourage it, but Windows would be extremely useful for some clients.

Reply:
Definetely.
I think that requires more gymnastics though, because the jaxlib package on conda-forge doesn't support windows at the moment. I guess this could be fixed by specifying the conda-forge jaxlib package only for non-windows, and adding a pypi jax dependency on windows. And I guess just don't allow the cuda12 feature on windows?